lib/px/service/kinesis/config.rb

```diff
@@ -2,12 +2,13 @@ module Px
   module Service
     module Kinesis
       class << self
-        DefaultConfig = Struct.new(:region, :credentials, :shard_count, :partition_key, :logger) do
+        DefaultConfig = Struct.new(:region, :credentials, :shard_count, :partition_key, :dev_queue_key, :logger) do
           def initialize
             self.region = AWS_DEFAULT_REGION
             self.shard_count = DEFAULT_SHARD_COUNT
             self.partition_key = BASE_PARTITION_KEY
             self.credentials = Aws::SharedCredentials.new
+            self.dev_queue_key = DEV_QUEUE_KEY
```

Comment:
Perhaps have this default to nil but set it in the host application in development mode, and if it's set, use that instead of Kinesis publish.